src/app/components/ThemeProvider/fontSizes.ts

```diff
@@ -1,13 +1,13 @@
 import pixelsToRem from '../../utilities/pixelsToRem';
-import { GROUP_B_MIN_WIDTH, GROUP_D_MIN_WIDTH } from './fontMediaQueries';
+import { GROUP_B_ONLY, GROUP_D_MIN_WIDTH } from './fontMediaQueries';
 import { GelFontSize, TypographyScript } from '../../models/types/theming';
 
 const getGelFontSizeStyles =
   (size: GelFontSize) => (script: TypographyScript) => ({
     fontSize: `${pixelsToRem(script[size].groupA.fontSize)}rem`,
     lineHeight: `${pixelsToRem(script[size].groupA.lineHeight)}rem`,
 
-    [GROUP_B_MIN_WIDTH]: {
+    [GROUP_B_ONLY]: {
```

Comment:
Please note that we found this bug during snapshot testing - the new snapshots did not match the old snapshots due to the incorrect MQ being used here. This will have affected a number of other snapshots - mainly for any new TS components  

This is to replicate the functionality in https://github.com/bbc/simorgh/blob/f80390e6b26e6e60a2ab082ac94917e41b87e410/src/app/legacy/psammead/gel-foundations/src/getTypeSizes.js#L25-L26